templates/shared_services/firewall/porter.yaml

```
@@ -1,7 +1,7 @@
 ---
 schemaVersion: 1.0.0
 name: tre-shared-service-firewall
-version: 0.8.0
+version: 0.8.1 # TODO: update before merge
```

keeping this open to remember

```suggestion
version: 1.0.0
```